modules/ting_search/includes/ting_search.admin.inc

```diff
@@ -43,8 +43,41 @@ function ting_search_admin_settings($form_state) {
   $form['ting_search']['ting_search_register_serie_title'] = array(
```

Comment:
Now we can probably remove the comment above.

Reply:
Also now we have 3 diffent fields dealing with similar functionality. Please consider wrapping them in a field set to improve the UI a bit.